ITunesLibraryParser/ITunesLibrary.cs

```diff
@@ -3,46 +3,143 @@
 using System.Linq;
 using System.Xml.Linq;
 
-namespace ITunesLibraryParser {
-  public interface IITunesLibrary {
-    IEnumerable<Track> Parse(string xmlFileLocation);
-  }
+namespace ITunesLibraryParser
+{
+    public class ITunesLibrary : IITunesLibrary
+    {
 
-  public class ITunesLibrary : IITunesLibrary {
+        private XDocument _xml;
+        private List<Track> _tracks;
 
-    public IEnumerable<Track> Parse(string fileLocation) {
-      var trackElements = LoadTrackElements(fileLocation);
-      return trackElements.Select(CreateTrack);
-    }
+        private Track CreateTrack(XElement trackElement)

```

Comment:
This object is getting large and is doing a few different things now. Not necessary for this PR, but I'd like to see the Track creation logic be pulled out into its own object.